images/cfw/usr/bin/heatbed_set.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+source /usr/bin/mqtt_access.sh
+
+function set_heatbed_temp() {
+    temp=$1
+    json="{ \
+        \"print\":{\"command\":\"gcode_line\", \"sequence_id\":\"2002\", \"param\":\"M140 S$temp\"} \
+    }"
+    mqtt_pub "$json"
+}
```

Comment:
Can we use `x1plus.client` commands to do this?  Or pull in a MQTT client into `x1plusd` (I think @jphannifan was talking about an asyncio wrapper for `paho`)?  I would like to avoid spawning shell scripts to the degree possible -- it seems like most of this stuff should really live in the `x1plus.client` library.